WRO-3879: Update node modules' lockfiles version (#5)
enactjs/eslint-config-enact-proxy: CHANGELOG.md

The following is a curated list of changes in the Enact proxy ESLint config:

## [unreleased]

### Fixed

> - Updated the version of package lock file to v2

Reviewer: ```suggestion
- Updated the version of npm-shrinkwrap file to v2
```

> - Updated the version of npm-shrinkwrap file to v2

Reviewer: ```suggestion
- Updated the `lockfileVersion` of npm-shrinkwrap file to v2
```

> - Updated the `lockfileVersion` of npm-shrinkwrap file to v2

Reviewer: ```suggestion
- Updated the `lockfileVersion` of npm-shrinkwrap file to v2.
```
